core/src/main/java/org/apache/accumulo/core/client/impl/BulkImport.java

```diff
@@ -252,7 +256,8 @@ KeyExtent lookup(Text row)
         throws IOException, AccumuloException, AccumuloSecurityException, TableNotFoundException;
   }
 
-  private static class ConcurrentKeyExtentCache implements KeyExtentCache {
+  @VisibleForTesting
+  static class ConcurrentKeyExtentCache implements KeyExtentCache {
```

Comment:
This cache class could be put in it's own file, especially since it has to be visible for testing anyway.